src/utils/Block.ts

```diff
@@ -0,0 +1,332 @@
+import { nanoid } from 'nanoid';
+import { EventBus } from './EventBus';
+import refElementsCollection from './RefElementsCollection';
+
+// Нельзя создавать экземпляр данного класса
+class Block {
+  static EVENTS = {
+    INIT: 'init',
+    FLOW_CDM: 'flow:component-did-mount',
+    FLOW_CDU: 'flow:component-did-update',
+    FLOW_RENDER: 'flow:render',
+  };
+
+  public readonly id = nanoid(6); // идентификатор блока
+
+  protected props: Record<string, any>; // параметры для блока
```

Comment:
Можно ли тут использовать unknown?
```suggestion
  protected props: Record<string, unknown>; // параметры для блока
```

Reply:
к сожалению насколько я понял в этом месте только с any работает, нам даже про это на вебинары говорили